dotcom-rendering/src/components/SignInGateSelector.importable.tsx

```diff
@@ -537,12 +544,24 @@ const SignInGateSelectorAuxia = ({
 		return null;
 	}
 
+	const ctaUrlParams = {
+		pageId,
+		host,
+		pageViewId,
+		idUrl,
+		currentTest: abTest,
+		componentId: abTest.id,
+	} satisfies Parameters<typeof generateGatewayUrl>[1];
```

Comment:
fancy!

Reply:
I seriously wish I were that good, but I just copied code from the original gate 😅

Reply:
Hah! And perhaps it should just be an `interface` instead...